angular-ngrx-scss/src/app/repository/components/repo-header/repo-navigation/repo-navigation.component.scss

```diff
@@ -0,0 +1,63 @@
+@use '../../../../shared/styles/variables.scss';
+
+.container {
+  max-width: 1536px;
+  margin: 0 auto 0 0;
+  nav {
+    margin-bottom: -1px;
+    display: flex;
+    ul {
+      list-style: none;
+      padding: 0;
+      margin: 0;
+
+      a {
+        display: inline-flex;
+        align-items: center;
+        text-decoration: none;
+        font-size: 1rem;
+        padding: 16px;
+        color: variables.$gray600;
+        &.active-tab {
+          border-bottom: 2px solid rgba(245, 158, 11, 1);
+
+          span {
+            font-weight: 600;
+            color: black;
+
+            &.icon {
+              color: variables.$gray600;
+            }
+          }
+        }
+        &:hover {
```

Comment:
General comment: When you hover a tab, the repo content jumps 1px down (probably some box-sizing: border-box; needs to be set somewhere). 🙂